sonar-dotnet-tests-library/src/main/java/org/sonar/plugins/dotnet/tests/CoverageReportImportSensor.java

```diff
@@ -82,28 +82,39 @@ public void execute(SensorContext context) {
   void analyze(SensorContext context, Coverage coverage) {
     coverageAggregator.aggregate(wildcardPatternFileProvider, coverage);
 
+    boolean hasAnyMainFileCovered = false;
     for (String filePath : coverage.files()) {
       FilePredicates p = context.fileSystem().predicates();
-      InputFile inputFile = context.fileSystem().inputFile(p.and(p.hasType(Type.MAIN), p.hasAbsolutePath(filePath)));
+      InputFile inputFile = context.fileSystem().inputFile(p.and(p.hasAbsolutePath(filePath)));
 
       if (inputFile == null) {
-        LOG.debug("Code coverage will not be imported for the following file outside of SonarQube: " + filePath);
+        LOG.debug("The file '" + filePath +"' is either excluded or outside of your solution folder therefore Code "
+         + "Coverage will not be imported.");
         continue;
       }
 
+      if (inputFile.type().equals(Type.TEST)) {
+        continue; // Do not log for test files to avoid pointless noise
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this trailing comment on the previous empty line. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3ATrailingCommentCheck)